openmrs/src/test/java/org/motechproject/openmrs/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -134,6 +135,9 @@ public void shouldCreateEncounterWithGivenParameters() {
         verify(encounterService).createEncounter(eq(CONFIG_NAME), encounterCaptor.capture());
 
         assertEquals(encounter, encounterCaptor.getValue());
```

Comment:
this equal is now redundant, just check if action returns proper encounter object